test/test_autograd.py

```diff
@@ -2792,14 +2808,38 @@ def forward(x):
         start_order = [
             'profiler::_record_function_enter',
             'outer',
+            'is_complex',
+            'mul',
             'mul',
+            'to',
+            'empty_strided',
+            'copy_',
+            'empty',
+            'is_complex',
             'add',
+            'add',
+            'to',
+            'empty_strided',
+            'copy_',
+            'empty',
             'profiler::_record_function_enter',
             'inner',
+            'is_complex',
+            'sub',
             'sub',
+            'to',
+            'empty_strided',
+            'copy_',
+            'empty',
             'profiler::_record_function_exit',
             'profiler::_record_function_exit',
+            'is_complex',
+            'div',
             'div',
+            'to',
+            'empty_strided',
+            'copy_',
+            'empty'
```

Comment:
We should seriously consider putting this in an expect test; now that we are profiling a lot more, it seems likely that this will wobble as people make small implementation changes.

Reply:
Or maybe we shouldn't match everything, just a subset of the "most important" op calls.